src/css/fonts.css

```diff
@@ -0,0 +1,102 @@
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,200i,300,300i,400,400i,600,600i,700,700i,900,900i");
+@import url("https://fonts.googleapis.com/css?family=Source+Serif+Pro:400,600,700");
+@import url("https://fonts.googleapis.com/css?family=Roboto+Slab:300,400,700");
+@import url("https://fonts.googleapis.com/css?family=Kalam:300,400,700");
+@import url("https://fonts.googleapis.com/css?family=Noto+Sans:400,400i,700,700i&subset=devanagari");
+
+/** Import Font Awesome Fonts */
+@font-face {
+  font-family: 'Font Awesome 5 Free';
+  font-style: normal;
+  font-weight: 400;
+  font-display: auto;
+  src: url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.eot");
+  src: url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.eot?#iefix") format("embedded-opentype"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff2") format("woff2"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff") format("woff"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.ttf") format("truetype"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.svg#fontawesome") format("svg");
+}
+
+@font-face {
+  font-family: 'Font Awesome 5 Brands';
+  font-style: normal;
+  font-weight: 400;
+  font-display: auto;
+  src: url("~@fortawesome/fontawesome-free/webfonts/fa-brands-400.eot");
+  src: url("~@fortawesome/fontawesome-free/webfonts/fa-brands-400.eot?#iefix") format("embedded-opentype"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-brands-400.woff2") format("woff2"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-brands-400.woff") format("woff"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-brands-400.ttf") format("truetype"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-brands-400.svg#fontawesome") format("svg");
+}
+
+/** Import Stanford Hosted Fonts **/
+@font-face {
```

Comment:
Good news is that these Source Serif Pro variants is now available on Google Fonts so we can just add that to the top Google fonts import line - they might not have the font weight 200 for this family (and we probably shouldn't use for a11y reasons too).